src/prefect/flows.py

```diff
@@ -236,6 +239,8 @@ def with_options(
             persist_result: A new option for enabling or disabling result persistence.
             result_storage: A new storage type to use for results.
             result_serializer: A new serializer to use for results.
+            cache_result_in_memory: A new toggle indicating if the flow's result should
```

Comment:
Important to note "new" here? Seems like this could become stale.

Reply:
This is in the `with_options` section for overriding settings on a new copy
```suggestion
            cache_result_in_memory: A new value indicating if the flow's result should
```